src/index.js

```diff
@@ -6,82 +6,44 @@ import $ from 'jquery'
 
```

Comment:
Changed to server-side rendering and eslinted the whole file.

Reply:
lol, rip my semicolons